File: scratch/fix_settings.py
import re

with open('lib/screens/dashboard/tabs/settings_tab.dart', 'r', encoding='utf-8') as f:
    content = f.read()

# Replace withOpacity
content = re.sub(r'\.withOpacity\((.*?)\)', r'.withValues(alpha: \1)', content)

# Replace activeColor
content = content.replace('activeColor:', 'activeThumbColor:')

# Replace context.mounted
content = content.replace('!context.mounted', '!mounted')

with open('lib/screens/dashboard/tabs/settings_tab.dart', 'w', encoding='utf-8') as f:
    f.write(content)

File: scratch/fix_settings.dart
import 'dart:io';

void main() {
  final file = File('lib/screens/dashboard/tabs/settings_tab.dart');
  var content = file.readAsStringSync();

  // Replace withOpacity
  content = content.replaceAllMapped(
      RegExp(r'\.withOpacity\((.*?)\)'),
      (match) => '.withValues(alpha: ${match.group(1)})');

  // Replace activeColor
  content = content.replaceAll('activeColor:', 'activeThumbColor:');

  // Replace context.mounted
  content = content.replaceAll('!context.mounted', '!mounted');

  file.writeAsStringSync(content);
}
